.circleci/config.yml

```diff
@@ -21,6 +21,11 @@ jobs:
           name: Install nfpm, rpmbuild
           command: |
             sudo make -f Makefile.tools nfpm-debian rpmbuild-debian
+      - run:
```

Comment:
Based on the latest test failure, looks like we need it for tests too. I should have realized that.

Reply:
thanks, it's fixed now